accounts/abi/bind/bind_test.go

```diff
@@ -1785,6 +1785,52 @@ var bindTests = []struct {
 		nil,
 		nil,
 	},
+	// Test errors introduced in v0.8.4
+	{
+		`NewErrors`,
+		`
+		pragma solidity >0.8.4;
+	
+		contract Test {
+			error MyError(uint256);
+			error MyError1(uint256);
+			error MyError2(uint256, uint256);
+			error MyError3(uint256 a, uint256 b, uint256 c);
+		}
+	   `,
+		[]string{"0x6080604052348015600f57600080fd5b50603f80601d6000396000f3fe6080604052600080fdfea2646970667358221220313fc94775b67962460298a5e968a46aed3683e1162a713923677e23373efa6364736f6c63430008060033"},
```

Comment:
Can you try to call the function with the throw error and decode it(with the generated binding)?